src/lib/components/pages/TenantNewEntity.vue

```diff
@@ -98,6 +109,13 @@ export default {
       console.log("this.$route.params : ", this.$route.params);
       return this.$route.params.clientId;
     },
+    parentId(){
+      console.log("this.$route.params: ", this.$route.params);
+      if(this.$route.params.entityId !== undefined){
```

Comment:
Just returning the `this.$route.params.entityId` is probably sufficient here.

Reply:
Agreed.